package.json

```diff
@@ -13,10 +13,12 @@
   "scripts": {
     "build": "tsc",
     "build:serverless": "ncc build src/index.ts -o dist",
-    "build:ci": "ncc build adapters/index.ts -o dist && mv ./dist/index.js ./bin",
+    "build:ci": "ncc build adapters/github-actions.ts -o dist",
+    "postbuild:ci": "mv ./dist/index.js ./bin",
```

Comment:
The `bin` folder is removed so `mv ./dist/index.js ./bin` will move `index.js` to the `bin` **file** instead of the `bin` folder. So in order to keep the `bin` folder indexed we could add an empty `.gitkeep` file to the `bin` folder

Reply:
I decided to build to project root and run from there instead.